client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -479,15 +486,15 @@ export class SeoForm extends React.Component {
 }
 
 const mapStateToProps = ( state ) => {
-	const selectedSite = getSelectedSite( state );
+	const site = getSelectedSite( state );
 	const siteId = getSelectedSiteId( state );
 	const siteIsJetpack = isJetpackSite( state, siteId );
 	// SEO Tools are available with Business plan on WordPress.com, and
 	// will soon be available on all Jetpack sites, so we're checking
 	// the availability of the module.
 	const isAdvancedSeoEligible =
-		selectedSite.plan &&
-		hasSiteSeoFeature( selectedSite ) &&
+		site.plan &&
```

Comment:
Maybe it's good to short circuit the check early, have we implicitly got this check in `hasSiteSeoFeature`?

Reply:
Yeah I think we do, I removed that check here: https://github.com/Automattic/wp-calypso/pull/53196/commits/6c722ae335fe3176cbfe6cd5c57698efe9b0f536

I also reverted the `selectedSite` -> `site` change since we're no longer using `site` with the selectors.